### ISSUE TITLE
E-Mail on Staff success page

### ISSUE THREAD

**Issue by ckleemann** [d120/pyophase#12]
When a Person registers as Staff on the success page also a e-mail should be send to the persons e-mail address

**Comment by exploide** [comment]
Don't know if this is still necessary since we put more information on the registration success page. But we can think about this once the production server got access to the mail server.

**Comment by max-weller** [comment]
Soll ich [sSMTP](https://wiki.debian.org/sSMTP) oder was vergleichbares draufziehen? Der Mailserver wird ja eh auf eine andere VM kommen

**Comment by exploide** [comment]
Ist ja aktuell nicht dringend, von daher muss das nicht jetzt sein. Von der ganzen anderen Software die auf der Kiste laufen wird, wird auch einiges E-Mail benötigen. Ist die Frage was da ingesamt ne gute Lösung wäre. Für dieses Ticket hier ist es jetzt nicht dringend.

**Comment by jlauinger** [comment]
Additionally to the one-way-confirmation of sending an email on successful registration, this email should contain a link to verify the email, and an exclamation mark sign should appear next to the person in the admin interface until its email is verified.

This should avoid the problem of typos in the email and therefore lack of being subscribed to Ophase mailing lists and reveiving important information. This has happened.

**Comment by ckleemann** [comment]
What @jlauinger wrote is a more complex issue. We should open another issue for it.

**Comment by exploide** [comment]
This issue is no longer blocked by missing mail functionality since 25b37e9ddd474d93a0aaa2cf15ab61cd4c75eebc includes all relevant settings and 024f4683a1f515543db6b312cf514c152a794f2c gives an example on how to send emails.
